recipes/libtins/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "libtins" %}
+{% set version = "4.2" %}
+{% set sha256 = "a9fed73e13f06b06a4857d342bb30815fa8c359d00bd69547e567eecbbb4c3a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mfontanini/{{ name|lower }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libtins', max_pin='x') }}
+  skip: true  # [win and vc<14]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+    - make  # [ unix ]
+    - libpcap
+
+test:
+  commands:
+    - test -f $PREFIX/include/tins/tins.h  # [not win]
```

Comment:
```suggestion
    - test -f $PREFIX/include/tins/tins.h  # [not win]
    - test -f $PREFIX/lib/libtins${SHLIB_EXT}  # [unix]
    - if not exist %PREFIX%\Library\bin\tins.dll exit 1  # [win]
```

Reply:
fixed.

Reply:
Please also add the line for Windows I mentioned here.

Reply:
fixed.